test/prettier.js

```diff
@@ -0,0 +1,92 @@
+#!/usr/bin/env node
+/* eslint-disable node/shebang */
+
+/**
+ * @fileoverview Runs `prettier` as an ESLint rule.
+ * @author Andres Suarez
+ */
+
+'use strict';
+
+// This test is optimized for debuggability.
+// Please do not attempt to DRY this file or dynamically load the fixtures.
+
+// ------------------------------------------------------------------------------
+// Requirements
+// ------------------------------------------------------------------------------
+
+const fs = require('fs');
+const path = require('path');
+
+const rule = require('..').rules.prettier;
+const RuleTester = require('eslint').RuleTester;
+
+// ------------------------------------------------------------------------------
+// Tests
+// ------------------------------------------------------------------------------
+
+const ruleTester = new RuleTester();
+
+ruleTester.run('prettier', rule, {
+  valid: [
+    // Correct style.
+    { code: '"";\n' },
+    // No pragma = No prettier check.
+    { code: '""\n', options: [null, '@format'] },
+    // Facebook style uses single quotes.
+    { code: `'';\n`, options: ['fb'] },
+    // Facebook style but missing pragma.
+    { code: `"";\n`, options: ['fb', '@format'] },
+    // Facebook style with pragma.
+    { code: `/** @format */\n'';\n`, options: ['fb', '@format'] }
+  ],
+  invalid: [
+    '01',
```

Comment:
This isn't a strong objection, but I have doubts about whether putting the invalid cases in separate files actually improves debuggability. If a single test case is failing, it seems like it would be easier to search for the corresponding code within a single file rather than opening/closing a bunch of test files to try to find the right one.